websocket/src/websocket.cpp

```diff
@@ -449,6 +462,68 @@ HandshakeResponse::~HandshakeResponse()
 }
 
 
+
+// ***************************************************************************************************
+// Emscripten Websocket library callbacks
+
+#if defined(__EMSCRIPTEN__)
+
+static WebsocketConnection* FindConnectionForEmscriptenWebSocket(EMSCRIPTEN_WEBSOCKET_T ws)
+{
+    for (int i = 0; i < g_Websocket.m_Connections.Size(); ++i )
+    {
+        if (g_Websocket.m_Connections[i]->m_WS == ws)
+            return g_Websocket.m_Connections[i];
+    }
+    return 0;
+}
```

Comment:
Shouldn't be needed if we're using userData.

Reply:
Yep, using userData!